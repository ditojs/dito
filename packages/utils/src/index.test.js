import * as index from './index.js'

describe('index', () => {
  it('exports all symbols', () => {
    expect(index).toMatchSnapshot()
  })
})
