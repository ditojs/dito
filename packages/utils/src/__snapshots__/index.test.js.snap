// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index exports all symbols 1`] = `
Object {
  "asArray": [Function],
  "asFunction": [Function],
  "asObject": [Function],
  "camelize": [Function],
  "capitalize": [Function],
  "clone": [Function],
  "debounce": [Function],
  "debounceAsync": [Function],
  "decamelize": [Function],
  "deepMerge": [Function],
  "defaultDateFormat": Object {
    "day": "numeric",
    "month": "long",
    "year": "numeric",
  },
  "defaultTimeFormat": Object {
    "hour": "2-digit",
    "minute": "2-digit",
    "second": "2-digit",
  },
  "deindent": [Function],
  "equals": [Function],
  "escapeHtml": [Function],
  "flatten": [Function],
  "formatDate": [Function],
  "getCommonOffset": [Function],
  "getCommonPrefix": [Function],
  "getEntriesAtDataPath": [Function],
  "getValueAtDataPath": [Function],
  "groupBy": [Function],
  "hyphenate": [Function],
  "is": [Function],
  "isAbsoluteUrl": [Function],
  "isArray": [Function],
  "isArrayLike": [Function],
  "isAsync": [Function],
  "isBoolean": [Function],
  "isCreditCard": [Function],
  "isDate": [Function],
  "isEmail": [Function],
  "isEmpty": [Function],
  "isFunction": [Function],
  "isHostname": [Function],
  "isInteger": [Function],
  "isNumber": [Function],
  "isObject": [Function],
  "isPlainObject": [Function],
  "isPromise": [Function],
  "isRegExp": [Function],
  "isString": [Function],
  "isUrl": [Function],
  "labelize": [Function],
  "mapKeys": [Function],
  "mapValues": [Function],
  "merge": [Function],
  "normalizeDataPath": [Function],
  "parseDataPath": [Function],
  "pick": [Function],
  "pickBy": [Function],
  "setDataPathEntries": [Function],
  "setValueAtDataPath": [Function],
  "shuffle": [Function],
  "stripTags": [Function],
  "toAsync": [Function],
  "toCallback": [Function],
  "toPromiseCallback": [Function],
  "underscore": [Function],
}
`;
